Makefile

```diff
@@ -26,9 +30,23 @@ check-license:
 
 tools: $(TOOLS)
 
+man: $(MAN)
+
+all: $(TOOLS) $(MAN)
+
 $(TOOLS): oci-%:
 	go build -ldflags "-X main.gitCommit=${COMMIT}" ./cmd/$@
 
+$(MAN): %.1: cmd/*/*.1.md
```

Comment:
This does not work (see [here][1], where *all* of your man pages are built from `oci-unpack.1.md`).  You want to use secondary expansion (with the line I suggested [here][2]).

[1]: https://github.com/opencontainers/image-tools/pull/92#issuecomment-264414166
[2]: https://github.com/opencontainers/image-tools/pull/92#discussion_r90596743